<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoorPrize</class>
 <widget class="QMainWindow" name="DoorPrize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoorPrize</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="door1">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="door2">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="door3">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>419</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Pick me instead!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>DOOR 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="image2Text">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>480</y>
      <width>280</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image3Text">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>480</y>
      <width>280</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image3">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>419</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;No, pick me!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>DOOR 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>419</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Pick me!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>DOOR 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>87</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image1Text">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>480</y>
      <width>280</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
